test/lib/y2firewall/clients/auto_test.rb

```diff
@@ -229,6 +228,82 @@
     end
   end
 
+  describe "#change" do
+    let(:result) { :ok }
+    let(:already_read) { false }
+    let(:exported_profile) { double("Exported Profile") }
+    let(:main_dialog) { instance_double("Dialog::Main", run: result) }
+
+    before do
+      allow(subject).to receive(:import)
+      allow(subject).to receive(:export).and_return(exported_profile)
+      allow(Y2Firewall::Dialogs::Main).to receive(:new).and_return(main_dialog)
+      allow(firewalld).to receive(:read?).and_return(already_read)
+    end
+
+    context "when the configuration is accepted" do
```

Comment:
Mmm it says when it is accepted but the results seems to be the rejected ones

Reply:
Oops! Wrong sentence. It seems that, accidentally, I switched them.

Thanks!